build_in_functions_task_filter_func.py

```diff
@@ -0,0 +1,27 @@
+# Implement your realization of the function filter
+from typing import Callable
+
+filtered_data = []
+
+
+def filter_for_allowed_types(iterable):
+    for data in iterable:
+        allowed_types = isinstance(data, (str, int, float))
+        if allowed_types:
+            filtered_data.append(data)
+
+
+def custom_filter(callback: Callable, iterable):
+    """
+    Custom filter func which process data
+    :param callback: allows to call needed func with diff types
+    :param iterable: data which needs to be filtered
+    """
+    return callback(iterable)
```

Comment:
it is not filtering. You just add all data together. What if I want to get only numbers that are> 5 from iterable. It will not work